### ISSUE TITLE
Group Module invite friends checkbox ids

### ISSUE THREAD

**Issue by jlopez3573** [SocialEngine/phpv4-issues#246]
In the "Group" module there is a "Invite.php" form with a field "users" (checkbox list).
this field creates a list of user's friends with id and name, but the id is wrong because in the "MemberController.php" method "inviteAction()" there is a wrong use on the PHP's "sort()" function because this reset the friend id. The best opcion is to use PHP's "asort()" function

this is the actual code:

```
$count = 0;
    $multiOptions = array();
    foreach( $friends as $friend )
    {
      if( $group->membership()->isMember($friend, null) ) continue;
      $multiOptions[$friend->getIdentity()] = $friend->getTitle();
      $count++;
    }
    sort($multiOptions);
    $form->users->addMultiOptions($multiOptions);
    $this->view->count = $count;
```

this will be the best option to use:

```
$count = 0;
    $multiOptions = array();
    foreach( $friends as $friend )
    {
      if( $group->membership()->isMember($friend, null) ) continue;
      $multiOptions[$friend->getIdentity()] = $friend->getTitle();
      $count++;
    }
    asort($multiOptions);
    $form->users->addMultiOptions($multiOptions);
    $this->view->count = $count;
```

or you should use this code too:

```
$count = 0;
    foreach( $friends as $friend )
    {
      if( $group->membership()->isMember($friend, null) ) continue;
      $form->users->addMultiOption($friend->getIdentity(), $friend->getTitle());
      $count++;
    }
    $this->view->count = $count;
```

**Comment by Anshuse** [comment]
Hi @jlopez3573 ,

Thanks for your efforts! This issue is already in our to-do list and will resolve in our next release.

Thanks!

**Comment by DonnaScriptTechs** [comment]
This is fixed for 4.9.0 when it is released. If you still have this issue after we release, please open a new bug report.

**Comment by Elshara** [comment]
Awesome!
